Script.js

```diff
@@ -0,0 +1,73 @@
+const logo = document.querySelector('.menuButton');
+const xButons = document.querySelector('.crossButton');
+const menu = document.querySelector('.popupContainer');
+
+logo.addEventListener('click', () => {
+  menu.style.display = 'flex';
+});
+
+xButons.addEventListener('click', () => {
+  menu.style.display = 'none';
+});
+
+document.querySelectorAll('.popupButtons').forEach((n) => n.addEventListener('click', () => {
+  menu.style.display = 'none';
+}));
+
+const profileCards = [
+  {
+    picture: '/img/pictures/dude1.png',
+    cardTitle: 'Lionel Messi',
+    cardSubtitle: 'Qatar 2022 WC champion',
+    cardsP: 'Latest WC champion, lead Argentina to won the 3rd title',
+    divClass: 'div1 cardTrans',
+  },
+  {
+    picture: '/img/pictures/dude2.png',
+    cardTitle: 'Andres Iniesta',
+    cardSubtitle: 'South Africa 2010 WC champion',
+    cardsP: 'Captain of Spain, lead his team to win their first WC',
+    divClass: 'div2 cardTrans',
+  }, {
+    picture: '/img/pictures/dude3.png',
+    cardTitle: 'Daniele De Rossi',
+    cardSubtitle: 'Germany 2006 WC champion',
+    cardsP: 'A great midfielder, lead Italy to won their 4th WC ',
+    divClass: 'div3 cardTrans',
+  }, {
+    picture: '/img/pictures/dude4.png',
+    cardTitle: 'Cafu (Marcos E. de Morais)',
+    cardSubtitle: 'Korea-Japan 2002 WC champion',
+    cardsP: 'Captain o Brazil, lead his team to won their 5th world cup',
+    divClass: 'div4 cardTrans',
+  }, {
+    picture: '/img/pictures/dude5.png',
+    cardTitle: 'Zinedine Zidane',
+    cardSubtitle: 'France 1998 WC champion',
+    cardsP: 'Captain of France, lead his team to won their first WC',
+    divClass: 'div5 cardTrans',
+  }, {
+    picture: '/img/pictures/dude6.png',
+    cardTitle: 'Miroslav Klose',
+    cardSubtitle: 'Brazil 2014 WC champion',
+    cardsP: 'highest scorer of the WC, lead Germany to win their 4th title',
+    divClass: 'div6 cardTrans',
+  }];
+
+const fillGridContainer = document.querySelector('.gridContainer');
+
+document.addEventListener('DOMContentLoaded', () => {
+  for (let i = 0; i < 6; i += 1) {
+    const html = `<div class="${profileCards[i].divClass} cardContainer">
+    <div class="cardPictureContainer"><img class="cardPicture" src="${profileCards[i].picture}" alt=""></div>
+    <div>
+        <h2 class="cardTitle">${profileCards[i].cardTitle}</h2>
+        <h3 class="cardsSubtitle">${profileCards[i].cardSubtitle}</h3>
+        <img class="cardUnderline" src="img/text/grayunderline.svg" alt="underline">
+        <p class="cardsP">${profileCards[i].cardsP}</p>
```

Comment:
- [ ] On mobile, the cards for speakers should just be two, then the `MORE`button should render the rest. Kindly implement this feature as done on the design 👍 

## Design:
![image](https://user-images.githubusercontent.com/50990925/227754745-296731b9-93e5-4d35-93e4-96555efc2beb.png)

## Your design:
![image](https://user-images.githubusercontent.com/50990925/227754760-61dda1e2-6819-4f61-9ebb-8ab29b41eb68.png)

Reply:
Ok... Done!